javascript-timers/timeout.js

```diff
@@ -0,0 +1,7 @@
+var headingElement = document.querySelector('h1');
+
+function helloThere() {
+  headingElement.innerHTML = 'Hello There';
```

Comment:
It's generally a bad idea to set `innerHTML` directly (for both performance and security reasons). Please find another way to modify the text.

Reply:
![Screen Shot 2022-01-27 at 7 49 19 PM](https://user-images.githubusercontent.com/44829206/151484369-ed01cfb4-9fd2-4252-81b8-dab1e6794a78.png)